obal/data/module_utils/copr.py

```diff
@@ -10,13 +10,16 @@ def __init__(self, message, command):
         self.command = command
         super(CoprCliCommandError, self).__init__(message) #pylint: disable-all
 
-def copr_cli(command, executable=None):
+def copr_cli(command, executable=None, config_file=None):
     """
     Run a copr-cli command
     """
     if executable is None:
         executable = 'copr-cli'
 
+    if config_file:
+        command = ['--config', config_file] + command
```

Comment:
I wish `corp-cli` would support setting this via an environment variable, but as it does not, this looks fine.